src/util.ts

```diff
@@ -31,10 +31,12 @@ export function parseFilePrelude(text: string): Prelude {
 			ret.pkg = { start: i, end: i };
 		}
 		if (line.match(/^(\s)*import(\s)+\(/)) {
-			ret.imports.push({ kind: 'multi', start: i, end: -1 });
+			ret.imports.push({ kind: 'multi', start: i, end: -1, package: 'multi-na' });
 		}
 		if (line.match(/^(\s)*import(\s)+[^\(]/)) {
-			ret.imports.push({ kind: 'single', start: i, end: i });
+			// Pull the package name
+			let match = line.match(/import "(.+)"/);
```

Comment:
Need to make sure this never fails.  Probably better to just modify the `if` match to capture the name as well?  I ran into failures when a line looked like `import  "byte"` for example (two spaces instead of one).